packages/synapse-react-client/tsconfig.json

```diff
@@ -3,7 +3,9 @@
   "compilerOptions": {
     "rootDir": "src/lib",
     "outDir": "dist",
-    "module": "CommonJS",
+    "module": "esnext",
+    "target": "esnext",
+    "moduleResolution": "Node",
```

Comment:
Vite was failing to optimize/resolve SRC without this change. This change didn't seem to cause any problems with the apps, and SWC doesn't use this output, so let's make this change.